frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -272,7 +279,8 @@ public APIGatewayProxyResponseEvent handleRequest(
             return generateApiGatewayProxyResponse(200, startResponse);
 
         } catch (JsonException e) {
-            return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1001);
+            LOG.error("Unable to serialize start response", e);
+            throw new RuntimeException(e);
```

Comment:
We'd like to stop throwing RTEs back to the Lambda runtime as it can cause cold starts for future invocations which will have an impact on performance.

Reply:
Very interesting, should I throw the JsonException instead? Otherwise I'm not really sure the best way to handle this, maybe return a 500?

Reply:
This has happened because we've received some bad json and it is something caused by the client so returning a 400 makes sense to me in this case.

Reply:
Cool have updated this now not to throw a RTE